inspirehep/modules/workflows/views.py

```diff
@@ -745,6 +763,30 @@ def start_author_workflow():
     return jsonify({'workflow_object_id': workflow_object_id})
 
 
+@workflow_blueprint.route('/literature', methods=['POST'])
```

Comment:
we should add @require_api_auth here as well

Reply:
I'll create US for this